### DISCUSSION TITLE
Problems in my student developer pack

### DISCUSSION THREAD

**Discussion by Kabirjafri** [community/community#109328] [GitHub Education] [answered]
### Hi everyone,

I have applied for my student developer pack and it got approved also but i am not getting any benefits . Kindly look into this problem and resolve it.
![Screenshot_20240220_103712_Chrome](https://github.com/community/community/assets/150506156/cb6b9222-224a-4263-9c9d-ac1418094169)

**Comment by namith1003** [comment, marked as answer]
This issue was the same for me, this is very misleading it should not say "Approved" but is more like "Pending", cause as it states below the benifits will be available soon, if you go to another educational pack page there is a snack bar stating that the applcation is still under processing and may take upto a week to process and the benfits to be awarded.

You most likely by the time i answer this already have the benfits available but for any other person reading this may be very helpful incase they see this discussion thread as you havent yet updated it im also assumin it is all ok now.

Hopefully, this answers anyone elses questions and feel free to leave a reply if anything,

A.A Namith Nimlaka

**Reply by Minakshig154** [reply]
Same for me ,anyone from GitHub care to review? Neither there is any contact email

**Reply by namith1003** [reply]
If you want a support/contact link there is one:

https://support.github.com/contact/education

This is for the educational support contact.
